modules/ROOT/pages/changelog.adoc

```diff
@@ -4,6 +4,17 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.4.2 - 2023-04-26
+
+=== Fixed
+
+* Selection was not correctly scrolled horizontally into view when using the selection.scrollIntoView API.
+* Menu buttons would have the Tabstopping behaviour in toolbar.
+* The editor would display a notification error when it fails to retirieve a blob image uri.
+* Setting an invalid unit in the fontsizeinput would change it do the default value instead of reverting it back to the previous valid value.
```

Comment:
Maybe add backticks:

```suggestion
* Setting an invalid unit in the `fontsizeinput` would change it do the default value instead of reverting it back to the previous valid value.
```